Storage/SQL/SQLiteQueue.swift

```diff
@@ -18,7 +18,7 @@ open class SQLiteQueue: TabQueue {
     }
 
     fileprivate func factory(_ row: SDRow) -> ShareItem {
-        return ShareItem(url: row["url"] as! String, title: row["title"] as? String, favicon: nil)
+        return ShareItem(url: row["url"] as! String, title: row["title"] as? String)
```

Comment:
Q. As I recall we would store favicons in SQLite, did we get rid of that?

Reply:
Yes, we now store the favicon urls in just a basic data file, similar to how we store tabs today. We added an expiration mechanism so the file won't grow forever and should always be a manageable size. See FaviconURLCache for details.